### ISSUE TITLE
Non-consecutive participant indices not supported

### ISSUE THREAD

**Issue by moltaire** [glamlab/glambox#2]
Datasets with non-consecutive participant indices (e.g., 0, 1, 3, 4) are currently not supported and result in shape errors.

One fast way to prevent errors is to re-index, but this might be not ideal, because parameter estimates,  etc. are then associated with a new index that was different in the data. Full support of non-consecutive indices, however, require us to keep a mapping between indices and IDs and convert back and forth between the two formats, **throughout the code**.

For now, data going into the model should already be formatted with a consecutive participant index.

**Comment by athms** [comment]
Yes. I agree. Instead of re-mapping this, I would opt that the input data has to be formatted accordingly. We should probably insert a quick check for this, when building the GLAM model.
